testing/sdk_cmd.py

```diff
@@ -365,14 +366,28 @@ def _ssh(cmd: str, host: str, timeout_seconds: int, print_output: bool, check: b
         ]
     )
 
+    direct_args = " ".join(
```

Comment:
nit: should we skip joining the empty string if `SSH_KEY_FILE` is not specified?

Reply:
I personally like the way it is now because the code is cleaner. Doing what you suggest will require an extra if statement here, for the same result.

Reply:
you mean:
```suggestion
    direct_args = " ".join(str(s) if s for s in
```
But yes, it's not critical.

Reply:
(the first level of `[ ]` are not needed), but this is a bit verbose for no real additional value. Let's leve it as is.